tests/e2e/spec/gutenberg_ro.e2e.spec.js

```diff
@@ -178,7 +178,7 @@ function runTests (driver, modes) {
             // Loads the ZIM archive for the mode if the mode is not skipped
             it('Load Modern zim file', async function () {
                 if (!serviceWorkerAPI) {
-                    console.log('\x1b[33m%s\x1b[0m', '      Test skipped.');
+                    console.log('\x1b[33m%s\x1b[0m', '      Following test skipped:');
```

Comment:
This just clarifies which test is being skipped, as the layout made it a bit confusing.